src/quantum/tp6-64bit.js

```diff
@@ -0,0 +1,606 @@
+/* global fetch */
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import { parse } from 'query-string';
+import Dashboard from '../dashboard';
+import Perfherder from './perfherder';
+import Countdown from './countdown';
+import TelemetryContainer from '../telemetry/graph';
+import SETTINGS from '../settings';
+import { quantum64QueryParams, flowGraphProps, statusLabels } from './constants';
+import GraphContainer from '../components/graph-container';
+import { CONFIG } from './config';
+import wrapSectionComponentsWithErrorBoundaries from '../utils/componentEnhancers';
+import PerfherderGraphContainer from '../containers/PerfherderGraphContainer';
+
+export default class TP6_64 extends React.Component {
+  constructor(props) {
+    super(props);
+    document.body.classList.add('multipage');
+    this.fetchNotes();
+  }
+
+  state = {
+    notes: {},
+  };
+
+  async fetchNotes() {
```

Comment:
You can drop `fetchNotes` completely. We are dropping the support for the Google Spreadsheet.

Reply:
Done.  This whole page is not complete.